File: Business/Models/ApplicantDetails.cs
using GrantManagement.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Business.Models
{
    public class ApplicantDetails
    {
        public int ApplicantId { get; set; }
        public int? GrantId { get; set; }
        public int UserId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public DateTime? Dob { get; set; }
        public int? Country { get; set; }
        public int? State { get; set; }
        public bool? PhyDisabled { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string PostCode { get; set; }
        public decimal? Mobile { get; set; }
        public decimal? Phone { get; set; }

        public List<EducationalDetail> EducationalDetails { get; set; }
    }
}

File: GrantManagement2/ClientApp/src/app/Models/ApplicantDetails.ts
import { EducationalDetails } from "./EducationalDetails";

export interface IApplicantDetails {
  Email: string,
  Password: string,
  FirstName: string,
  LastName: string,

  ApplicantId: number,
  GrantId: number
  UserId: number,
  Dob: Date,

  Country: number,
  State: number,
  PhyDisabled: boolean,
   Address:string,
    City :string,
      PostCode :string,
     Mobile :number,
  ApplicationStatus: string,
  ReviewStatus: boolean,
     Phone:number,
  EducationalDetails: EducationalDetails[]


}

export class ApplicantDetails implements IApplicantDetails {
  Email: string;
  Password: string;
  FirstName: string;
  LastName: string;

  ApplicantId: number;
  GrantId: number;
  UserId: number;
  Dob: Date;

  Country: number;
  State: number;
  PhyDisabled: boolean;
  Address: string;
  City: string;
  PostCode: string;
  Mobile: number;
  Phone: number;
  ApplicationStatus: string;
  ReviewStatus: boolean;
  EducationalDetails: EducationalDetails[];

  constructor() {
    

  }

}
